microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md

```diff
@@ -28,18 +28,35 @@ Uploads a package to update an existing draft submission of an add-on product.
 |---|---|
 | `POST` | `/products/{productID}/submissions/draft/package` |
 
+
+<!-- ---------- -->
 ###### URI parameters
 
 | URI parameter | Description |
 |---|---|
 | `productID` | Required.  The product ID of the product to which the package must be uploaded. |
 
+
+<!-- ---------- -->
 ###### Request headers
 
-*  Required.  `Authorization: Bearer <auth token>`
+The following request headers are required:
+
+##### [v2](#tab/v2)
```

Comment:
if i were a developer submitting an update to my extension, should i use the v2 required request headers, or the v1 required request headers? 
![image](https://github.com/user-attachments/assets/cdc75daa-49a6-4ceb-a302-6922d066f13b)

not sure why we're showing & describing both v2 as "required" and v1 as "required".  
Do we need to introduce the concept of v2 and v1?
Do we need to show dates for v2 & v1?

@yvsubhash @dhruv16dhr 

rendered article:
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/api/addons-api-reference?branch=pr-en-us-3262&tabs=v2#request-headers